plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -798,6 +802,9 @@ int process_and_send(struct flb_cloudwatch *ctx, const char *input_plugin,
     int check = FLB_FALSE;
     int found = FLB_FALSE;
     struct flb_time tms;
+    struct mk_list *tmp;
+    struct mk_list *head;
+    struct flb_intermediate_metric *an_item;
```

Comment:
nit: can we put these after line 810? The comment at line 809, will make more sense.